packages/oc-template-react-compiler/lib/viewTemplate.js

```diff
@@ -8,12 +8,13 @@ const viewTemplate = ({
   var modelHTML =  model.__html ? model.__html : '';
   var staticPath = model.reactComponent.props._staticPath;
   var props = JSON.stringify(model.reactComponent.props);
+  var cssTag = '<style data-react-root="${reactRoot}">${css}</style>';
   return '<div id="${reactRoot}" class="${reactRoot}">' + modelHTML + '</div>' +
-    '<style>${css}</style>' +
     '<script>' +
     'window.oc = window.oc || {};' +
     'oc.cmd = oc.cmd || [];' +
     'oc.cmd.push(function(oc){' +
+      'oc.$(\\'' + cssTag + '\\').appendTo(document.head);' +
```

Comment:
I don't think is that necessary to rely on jquery for that. I'll suggest we try wherever is possible to avoid creating unneded dependencies

Reply:
That will become 8 lines of code that I was hoping to reduce from the component bundles...what about we wrap this into oc client?
`oc.appendStyleToHead('.css{}');`
So that I can keep this small and neat.

Reply:
Yes, I think that will be much better